src/runtime_src/core/edge/user/device_linux.cpp

```diff
@@ -233,4 +285,33 @@ write(uint64_t offset, const void* buf, uint64_t len) const
   throw error(-ENODEV, "write failed");
 }
 
+
+void 
+device_linux::
+reset(const char* subdev, const char* key, const char* value) const 
+{
+  int val = atoi(value);
+
+  switch(val) {
+  case 1:
+    throw error(-ENODEV, "Hot reset not supported");
+    break;
```

Comment:
Give me a `break` here ;-)
I guess you can remove all the breaks...

Reply:
haha :-)  For all but 5, yeah.

Reply:
Do not brake so hard! Yes, case 5 is a good br(e)aking point :-)

Reply:
reduced the 'g' force :)